pod.go

```diff
@@ -179,6 +201,31 @@ func (pc *podCollector) Collect(ch chan<- prometheus.Metric) {
 	}
 }
 
+func kubeLabelsToPrometheusLabels(labels map[string]string) ([]string, []string) {
+	labelKeys := make([]string, len(labels))
+	labelValues := make([]string, len(labels))
+	i := 0
+	for k, v := range labels {
+		labelKeys[i] = "label_" + sanitizeLabelName(k)
+		labelValues[i] = v
+		i++
+	}
+	return labelKeys, labelValues
+}
+
+func sanitizeLabelName(s string) string {
+	return invalidLabelCharRE.ReplaceAllString(s, "_")
+}
+
+func podLabelsDesc(labelKeys []string) *prometheus.Desc {
+	return prometheus.NewDesc(
```

Comment:
@brancz It seems that prometheus will use `fqName`(`descPodLabelsName`) and `help`(`descPodLabels`) to uniquely define an metric. Once we first register it, the labels can be modified on the fly. Am i right?